package.json

```diff
@@ -12,7 +12,7 @@
   ],
   "private": true,
   "version": "1.2.0",
-  "branch": "main",
+  "branch": "latest",
```

Comment:
I think to reduce confusion we should maintain consistency with the actual branch name and then just add a minor translation in the docs service to go from `main` to `latest`.